WordPress/Jetpack/Classes/ViewRelated/WordPress-to-Jetpack Migration/Common/Navigation/MigrationFlowCoordinator.swift

```diff
@@ -26,7 +26,7 @@ final class MigrationFlowCoordinator: ObservableObject {
 
         // Skip the migration if the user just created an account and haven't
         // created any site yet.
-        if BlogListDataSource().visibleBlogsCount == 0 {
+        if (try? BlogQuery().blogs(in: ContextManager.shared.mainContext))?.count == 0 {
```

Comment:
We are migrating _all_ sites, not just the visible ones. The "visible" feature was only designed for hiding sites from the picker.